<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaneDlgClass</class>
 <widget class="QWidget" name="PlaneDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>平面拟合</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QLineEdit" name="Iteration">
       <property name="text">
        <string>10000</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="ifReverse">
       <item>
        <property name="text">
         <string>默认</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>反向</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="Confidence">
       <property name="text">
        <string>0.99</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>  最大迭代次数：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="minPointNum">
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LocalPoint">
       <property name="text">
        <string>0.01</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>  置信度：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>  筛选条件(最低点数)：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>  过滤条件：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>  局内点与模型距离：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
	border-radius:8px;
  	text-align: center center;
} 

/*鼠标放在按钮上方*/
QPushButton:hover:enabled {
	background-color: rgba(255, 255, 255,150);
	border:2px outset rgba(100, 100, 100,150);
} 

/*鼠标点击按钮*/
QPushButton:pressed:enabled {
	background-color: rgba(255, 255, 255,255);
	border:4px outset rgba(100, 100, 100,255);
}

QPushButton:enabled{ 
	color: rgb(0, 0, 0);
	background-color: rgba(255, 255, 255,0);
	border:2px outset rgb(100, 100, 100);
}

QPushButton:disabled{ 
	color: rgb(150, 150, 150);
	background-color: rgba(150, 150, 150,0);
	border:2px outset rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
